nms/app/packages/magmalte/scripts/fuji-upgrade/pre-upgrade-migration.sh

```diff
@@ -36,19 +36,19 @@ PRE-REQUISITES
 EOF
 
 # Get the Magma k8s namespace
-read -p "Enter Kubernetes namespace [magma]: " magma_namespace
-magma_namespace=${magma_namespace:-magma}
+read -p "Enter Kubernetes namespace [orc8r]: " magma_namespace
```

Comment:
Minor nit: 
Can you add this  set -euf -o pipefail at top of the script here(https://sipb.mit.edu/doc/safe-shell/)